modules/book-collection.js

```diff
@@ -0,0 +1,56 @@
+import BookModel from './book-model.js';
+import { clearInputFields } from './change-navbar.js';
+
+class BookCollection {
+  constructor() {
+    this.allBooks = JSON.parse(localStorage.getItem('allBooks')) || [];
+    this.addButton = document.getElementById('add-btn');
+    this.addButton.addEventListener('click', this.addBook.bind(this));
+  }
+
+  addBook() {
+    const titleInput = document.getElementById('title');
+    const authorInput = document.getElementById('author');
+    const title = titleInput.value;
+    const authorName = authorInput.value;
+    if (title && authorName) {
+      const bookList = new BookModel(title, authorName);
+      this.allBooks.push(bookList);
+      this.saveCollection();
+      clearInputFields();
+      this.showBooks();
+    }
+  }
+
+  removeBook(title, author) {
+    this.allBooks = this.allBooks.filter(
+      (book) => book.title !== title || book.authorName !== author,
+    );
+    this.saveCollection();
+    this.showBooks();
+  }
+
```

Comment:
- [ OPTIONAL ] Great job working on the ES6 syntax but I have a small issue you can improve it, after adding two books with the same name of author and book when I try to remove one of them so both books are removing from the list so I think you should add an id to each book so whenever the user to try remove one just remove the book that he want to remove 💯🚀✨⭐ and I am assiging it as optional because it is not part of the requirements but it is very important to improve